### PR TITLE
Allow images to be hidden on mobile

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
This PR extends the Card api to support the use case where images might show on desktop but be hidden on mobile.

This is achieved here by making the `imagePositionOnMobile` prop required if you want the image to be displayed on mobile. If this prop is not provided then the image will be hidden.

## Why?
The `DynamicSlow` container is an example for when a card shows its image at larger breakpoints but not on mobile.

| Before      | After      |
|-------------|------------|
| Screenshot 2022-04-27 at 15 46 48 | Screenshot 2022-04-27 at 15 46 00 |

### CODE REVIEW

File: dotcom-rendering/src/web/components/DynamicFast.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ export const DynamicFast = ({ trails }: Props) => {
 						starRating={secondary.starRating}
 						branding={secondary.branding}
 						supportingContent={secondary.supportingContent}
+						imagePositionOnMobile="left"
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/DynamicFast.tsx:83]: Now that we need to specify this value I am explicitly adding it in lots of places

File: dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { until } from '@guardian/source-foundations';
 
 type Props = {
 	children: React.ReactNode;
-	imagePositionOnMobile: ImagePositionType;
+	imagePositionOnMobile?: ImagePositionType;
```
[the remark this answers is not in the document]
Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx:7]: Nice, I like it. This is more explicit. Now we can set a default too

[the remark this answers is not in the document]
Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx]: Done. This is so much nicer!

**Review by OllysCoding — APPROVED** [review]
+1 Looks good!

**Pull request merged**